packages/@glimmer/util/lib/list-utils.ts

```diff
@@ -96,20 +94,6 @@ export class LinkedList<T extends LinkedListNode> implements Slice<T>, Drop {
 
     return node;
   }
-
-  [WILL_DROP]() {
```

Comment:
I'm not sure we ever actually destroyed a LinkedList before, I couldn't find a place in the code. This seems like it was conceptually part of the original work, but then didn't get integrated in the end. I don't think it makes much sense anymore with the new direction we're going.